build/storage/tests/it/ptf_tests/tests/test_deploy_containers.py

```diff
@@ -5,58 +5,71 @@
 import os
 
 from ptf.base_tests import BaseTest
-from system_tools.const import STORAGE_DIR_PATH
-from tests.steps.initial_steps import (
-    CloneIPDKRepository,
-    RunHostTargetContainer,
-    RunIPUStorageContainer,
-    RunStorageTargetContainer,
-)
-from system_tools.test_platform import HostTargetPlatform, IPUStoragePlatform, StorageTargetPlatform
-from test_connection import BaseTerminalMixin
-
-
-class TestTestPlatforms(BaseTerminalMixin, BaseTest):
+from system_tools.test_platform import (HostTargetPlatform, IPUStoragePlatform,
+                                        StorageTargetPlatform)
+
+from tests.steps.initial import (CloneIPDKRepository, RunCMDSenderContainer,
+                                 RunHostTargetContainer,
+                                 RunIPUStorageContainer,
+                                 RunStorageTargetContainer)
+
+
+class TestTestPlatforms(BaseTest):
     def setUp(self):
-        super().setUp()
-        self.storage_target_platform = StorageTargetPlatform()
-        self.ipu_storage_platform = IPUStoragePlatform()
-        self.host_target_platform = HostTargetPlatform()
+        self.storage_target_platform = StorageTargetPlatform(False)
+        self.ipu_storage_platform = IPUStoragePlatform(False)
+        self.host_target_platform = HostTargetPlatform(False)
 
     def runTest(self):
         self.storage_target_platform.set_system_setup()
-        self.ipu_storage_platform.set_system_setup()
         self.host_target_platform.set_system_setup()
+        self.ipu_storage_platform.set_system_setup()
 
         self.storage_target_platform.check_system_setup()
         self.ipu_storage_platform.check_system_setup()
         self.host_target_platform.check_system_setup()
 
+    def tearDown(self):
+        pass
 
-class TestDeployContainers(BaseTerminalMixin, BaseTest):
+
+class TestDeployContainers(BaseTest):
     def setUp(self):
-        super().setUp()
-        self.storage_target_terminal.delete_all_containers()
-        self.ipu_storage_terminal.delete_all_containers()
-        self.host_target_terminal.delete_all_containers()
+        self.storage_target_platform = StorageTargetPlatform(False)
```

Comment:
can we replace this boolean flag with an enum `ContainerStartPolicy` and pass `ContainerStartPolicy.DO_NOT_START` here?
It is not obvious what False here means without digging into the code

Reply:
possibility to create platforms without containers deleted

Reply:
Yes, but to understand what that `False` means we need to go into corresponding class and figure that out.
What I am suggesting is to create an enum which will do the same thing as that flag, but that enum will provide intention revealing names